.gitignore

```diff
@@ -47,7 +47,6 @@ MigrationBackup/
 /docs/build
 /docs/_sources
 **/packages/*
-/Source/*/Properties/launchSettings.json
```

Comment:
Don't remove this.

Reply:
Why? The launchSettings.json file is required to launch a project with custom arguments in Visual Studio.

Reply:
Those file change based on local/individual settings and preferences. 

Since it's not universal, it should not be part of the project.

Reply:
However it creates issues like not being able to run a local project via Visual Studio.

Reply:
It can be overridden when necessary, like on the examples, but not the whole project.